src/extension/generalMobilePlatform.ts

```diff
@@ -81,31 +81,45 @@ export class GeneralMobilePlatform {
         return Q.resolve<void>(void 0);
     }
 
-    protected getOptFromRunArgs(optName: string, binary: boolean = false): any {
-        if (this.runArguments.length > 0) {
-            const optIdx = this.runArguments.indexOf(optName);
-            let result: any = false;
+    public static getOptFromRunArgs(runArguments: any[], optName: string, binary: boolean = false): any {
+        if (runArguments.length > 0) {
+            const optIdx = runArguments.indexOf(optName);
+            let result: any = undefined;
 
             if (optIdx > -1) {
-                result = binary ? true : this.runArguments[optIdx + 1];
+                result = binary ? true : runArguments[optIdx + 1];
             } else {
-                for (let i = 0; i < this.runArguments.length; i++) {
-                    const arg = this.runArguments[i];
+                for (let i = 0; i < runArguments.length; i++) {
+                    const arg = runArguments[i];
                     if (arg.indexOf(optName) > -1) {
-                        result = binary ? true : arg.split("=")[1].trim();
+                        if (binary) {
+                            result = true;
```

Comment:
Do we need to process binary arguments like this? It is not clear why this function returns found argument or true.

Reply:
@SounD120 I've made this like that due to nature of binary parameter. Please see comment here:
https://github.com/microsoft/vscode-react-native/pull/995/files#diff-1ed5a09eac4b721889c3f7a3ae220802R109